README.md

```diff
@@ -53,6 +59,37 @@ q: 58.84 kB (5.07%)
 <self>: 195.57 kB (16.9%)
 ````
 
+### Plugin Usage
+
+Install this libarary as a development dependency to your webpack project:
```

Comment:
Typo: "libarary". Could you also add an introductory sentence that explains what the tool does when you use it as a Webpack plugin (that is, outputs a list of dependencies sorted by size to a given file)?